config/initializers/datadog.rb

```diff
@@ -2,6 +2,7 @@
   c.env = Rails.env
   c.tracing.enabled = ENV["DD_API_KEY"].present?
   c.tracing.partial_flush.enabled = true
+  c.tracing.log_injection = !Rails.env.development?
```

Comment:
Nit: should this also check that we're not in unit tests?

Reply:
yeah actually I think the only environment we want this enabled is prod

Reply:
```suggestion
  c.tracing.log_injection = Rails.env.production?
```